cavalier_contours/src/macros.rs

```diff
@@ -25,9 +25,9 @@ macro_rules! assert_fuzzy_eq {
                 if !(left_val.fuzzy_eq_eps(*right_val, *eps_val)) {
                     panic!(
                         r#"assertion failed: `left.fuzzy_eq_eps(right, eps)`
-  left: `{:?}`,
- right: `{:?}`
- eps: `{:?}`"#,
+                          left: `{:?}`,
+                         right: `{:?}`
+                           eps: `{:?}`"#,
```

Comment:
Same as other place:
Don't think we want this indented as it's inside raw string literal so it will format weird (these lines indented significantly while others are not).